src/Frontend/RecompileFrontendAssets.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+/*
+ * This file is part of Flarum.
+ *
+ * (c) Toby Zerner <toby.zerner@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Flarum\Frontend;
+
+use Flarum\Extension\Event\Disabled;
+use Flarum\Extension\Event\Enabled;
+use Flarum\Foundation\ValidationException;
+use Flarum\Locale\LocaleManager;
+use Flarum\Settings\Event\Saved;
+use Flarum\Settings\Event\Saving;
+use Flarum\Settings\OverrideSettingsRepository;
+use Flarum\Settings\SettingsRepositoryInterface;
+use Illuminate\Contracts\Container\Container;
+use Illuminate\Contracts\Events\Dispatcher;
+use Illuminate\Filesystem\FilesystemAdapter;
+use League\Flysystem\Adapter\NullAdapter;
+use League\Flysystem\Filesystem;
+use Less_Exception_Parser;
+
+class RecompileFrontendAssets
+{
+    /**
+     * @var FrontendCompilerFactory
+     */
+    protected $assets;
+
+    /**
+     * @var LocaleManager
+     */
+    protected $locales;
+
+    /**
+     * @var Container
+     */
+    protected $container;
+
+    /**
+     * @param FrontendCompilerFactory $assets
+     * @param LocaleManager $locales
+     * @param Container $container
+     */
+    public function __construct(FrontendCompilerFactory $assets, LocaleManager $locales, Container $container)
+    {
+        $this->assets = $assets;
+        $this->locales = $locales;
+        $this->container = $container;
```

Comment:
This container is not being used in this class. If it's just used in the subclasses, just inject it there, please.